webpack/webpack.config.dev.js

```diff
@@ -75,7 +75,8 @@ export default {
     ]),
     new HappyPack({
       loaders: ['babel'],
-      threads: 4
+      threads: 4,
+      verbose: false
```

Comment:
it still drops 1 line to the console, but the fix is already on github & should be published to npm soon.